components/item/AwesomeSwiper.vue

```diff
@@ -91,6 +93,9 @@
     computed: {
       swiper() {
         return this.$refs.awesome.$swiper;
+      },
+      playableMedia() {
```

Comment:
This is only ever used by `isSinglePlayableMedia`, which no longer accepts an argument, so just replace this computed property with one for `isSinglePlayableMedia`.